database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -20,6 +20,7 @@ public function up()
             $table->unsignedTinyInteger('role')->default(\App\User::USER); // default role user
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->integer('role');
```

Comment:
What happens if the dev has already run the migrate file before you make changes?

Reply:
Yes, i deleted it, will used migrate dev